functions/venue.js

```diff
@@ -68,6 +68,8 @@ const PlacementState = {
   Hidden: "HIDDEN",
 };
 
+const UNCLICKABLE_ROOM_TYPE = "UNCLICKABLE";
```

Comment:
This should be implemented as an 'enum' in a similar war to `VenueTemplate`/etc.

See also: https://github.com/sparkletown/sparkle/pull/1305#issuecomment-813780291


```suggestion
// These represent all of our 'special' room types (they should remain alphabetically sorted) If no type is set, then it is just a 'normal room'.
// @debt unify this with RoomTypes in src/types/rooms.ts + share the same code between frontend/backend
const RoomType = {
  unclickable = "UNCLICKABLE",
}
```

(It would also be good to rename the existing `RoomTypes` enum in the frontend code to `RoomType`, as enums shouldn't be pluralised)

Reply:
Resolving this and the rest of them as the change-set has been reverted.